doc/quick-install.md

```diff
@@ -1,64 +1,89 @@
 # Jitsi Meet quick install
 
-This document describes the required steps for a quick Jitsi Meet installation on a Debian based GNU/Linux system. Debian 8 (Jessie) or later, and Ubuntu 14.04 or later are supported out-of-the-box.
+This guide helps you  ___host your own Jitsi server___. If you want to have a video conference without setting up any infrastructure, use https://meet.jit.si instead.
 
-Debian Wheezy and other older systems may require additional things to be done. Specifically for Wheezy, [libc needs to be updated](http://lists.jitsi.org/pipermail/users/2015-September/010064.html).
+## Requirements
 
-Also note that a recent default Ubuntu installation has only the `main` repository enabled, and Jitsi Meet needs packages from `universe`. Check your `/etc/apt/sources.list` file, and if `universe` is not present refer to [Ubuntu's documentation](https://help.ubuntu.com/community/Repositories/Ubuntu) on how to enable it. (Usually it amounts to copying the `main` lines and changing to `universe`.)
+The installation steps documented here are intended for Debian-based GNU/Linux systems.
 
-N.B.:
+Systems running the following versions are supported by this guide:
 
-a.) All commands are supposed to be run by root. If you are logged in as a regular user with sudo rights, please prepend ___sudo___ to each of the commands.
+1. Debian 8 (Jessie) or later
+1. Ubuntu 14.04 or later
```

Comment:
I have a question about these versions, I'm not sure about those. Is that something you tested or we have it stated somewhere?

Reply:
I've carried these over from the existing version in the quick-install guide ([diff ref](https://github.com/jitsi/jitsi-meet/pull/5227/files#diff-39d47d1432f7af0aa02f3e913097320aL3)), but no, I can't confirm testing of the instructions going back to those OS versions.

Reply:
Another small detail to add - during uninstall (not currently modified in this changeset), there were some slight issues around package name mismatches.

I'd considered trying to clean that section up, but due to concerns about exactly this kind of backwards-compatibility testing, I didn't feel confident in applying changes which only worked locally (Ubuntu 19.10).